comps/llms/text-generation/tgi/llm.py

```diff
@@ -45,6 +56,17 @@
 async def llm_generate(input: Union[LLMParamsDoc, ChatCompletionRequest, SearchedDoc]):
     if logflag:
         logger.info(input)
+
+    if input.model and MODEL_CONFIGS and configs_map:
+        if configs_map.get(input.model):
+            config = configs_map.get(input.model)
+            llm_endpoint = config.get("endpoint")
+        else:
+            logger.error(f"Input model {input.model} not present in model_configs")
+            raise ConfigError(f"Input model {input.model} not present in model_configs")
+
+    llm = AsyncInferenceClient(model=llm_endpoint, timeout=600)
```

Comment:
Please check the CI test error, looks like llm_endpoint is empty string for this case, please fix it.
![image](https://github.com/user-attachments/assets/58d740eb-da83-4685-a9fc-2e7f34c395e0)

Reply:
Thanks for the review. Fixed it.